zookeeper-server/src/test/java/org/apache/zookeeper/test/ClientSSLTest.java

```diff
@@ -147,6 +148,8 @@ public void testClientServerSSLWithCnxnClassName() throws Exception {
     @ParameterizedTest(name = "sslProvider={0}, fipsEnabled={1}, hostnameVerification={2}")
     @MethodSource("positiveTestData")
     public void testClientServerSSL_positive(SslProvider sslProvider, String fipsEnabled, String hostnameVerification) throws Exception {
+        //Skipping this test for s390x arch as netty-tc-native is not supported
+        assumeFalse(System.getProperty("os.arch").contains("s390x"), " Skipping for s390x arch as netty-tcnative is not yet supported.");
```

Comment:
this change is not related to this patch
can you move it to another patch please ?

Reply:
yes, it is required for each PRs, thanks